FULL_HELP_DOCS.md

```diff
@@ -117,10 +117,12 @@ Get Id of builtin Soroban Asset Contract. Deprecated, use `stellar contract id a
 * `--network-passphrase <NETWORK_PASSPHRASE>` — Network passphrase to sign the transaction sent to the rpc server
 * `--network <NETWORK>` — Name of network to use from config
 * `--source-account <SOURCE_ACCOUNT>` — Account that signs the final transaction. Alias `source`. Can be an identity (--source alice), a secret key (--source SC36…), or a seed phrase (--source "kite urban…")
-* `--hd-path <HD_PATH>` — If using a seed phrase, which hierarchical deterministic path to use, e.g. `m/44'/148'/{hd_path}`. Example: `--hd-path 1`. Default: `0`
 * `--global` — Use global config
 * `--config-dir <CONFIG_DIR>` — Location of config directory, default is "."
-* `--check` — Check with user before signature. Eventually this will be replaced with `--yes`, which does the opposite and will force a check without --yes
+* `--sign-with-key <SIGN_WITH_KEY>` — Sign with secret key
```

Comment:
I think we should clarify that it's the "entity" as in `--source-account` (i.e. `Can be an identity (--source alice), a secret key (--source SC36…), or a seed phrase (--source "kite urban…")`)